src/vm/vm_memory/memory.rs

```diff
@@ -65,6 +65,12 @@ impl Memory {
     }
 }
 
+impl Default for Memory {
```

Comment:
Is this really needed?

Reply:
Not Really.
It wasn't in my plans to add Memory and MemorySegmentManager Defaults. 
But the GitHub actions check `cargo clippy  -- -D warnings` throws an error if I didn't add them.